### PR TITLE
Embeds: Add filter for default attributes

### LINKED ISSUE #4982: Allow developers modify default attributes on embed block
## Feature Description

Add a filters to enable developers to modify default height / width of a emblocks block / shortcodes. This will help improve styling and theme integrations. 

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Implementation Brief

- Add a filter in the [default_attrs](https://github.com/google/web-stories-wp/blob/87048d9e224d82e015ef833bf0e28d3d7614b442/includes/Embed_Base.php#L84-L100) method. 
- Change [default params](https://github.com/google/web-stories-wp/blob/87048d9e224d82e015ef833bf0e28d3d7614b442/includes/Block/Embed_Block.php#L75-L97) on block. 
- Remove defaults [here](https://github.com/google/web-stories-wp/blob/main/assets/src/story-embed-block/block/block.json#L16-L20).
- Pass defaults to through in block [settings](https://github.com/google/web-stories-wp/blob/87048d9e224d82e015ef833bf0e28d3d7614b442/includes/Block/Embed_Block.php#L112-L116).

### PR DESCRIPTION
**Pull request by spacedmonkey**
## Summary

Allow developers modify default attributes on embed block

## Relevant Technical Choices

Values are now passed from PHP to React to allow for easier filtering by developers. 

## Testing Instructions

---

Fixes #4982

### CODE REVIEW

**Comment by swissspidy** [comment]
I think this already covered by 

https://developer.wordpress.org/block-editor/developers/filters/block-filters/#blocks-registerblocktype
https://developer.wordpress.org/block-editor/developers/filters/block-filters/

?

This approach here doesn‘t seem very common for blocks.

**Comment by spacedmonkey** [comment]
This way keeps the default width and height the same between shortcode and block.

**Comment by spacedmonkey** [comment]
It is was a little weird way of doing it. Do you think there is still value in filter [here](https://github.com/google/web-stories-wp/pull/4983/files#diff-ec08e321ea253fe7898c04a885c31add1c6336a361c55c5731e077cf5dcf94d2R108).

File: includes/Embed_Base.php [hunk 1]
```diff
@@ -113,14 +113,23 @@ public function filter_kses_allowed_html( $allowed_tags ) {
...
 		];
+
+		/**
+		 * Filters settings passed to the web stories embed.
+		 *
+		 * @since 1.1.0
+		 *
+		 * @param array $attrs Array of settings passed to web stories embed.
+		 */
+		return apply_filters( 'web_stories_embed_attributes', $attrs );
```
Comment (swissspidy) [inline includes/Embed_Base.php]:
```suggestion
		return apply_filters( 'web_stories_embed_default_attributes', $attrs );
```

**Review by swissspidy — APPROVED** [review]
This seems more reasonable. LGTM with 1 nit.

**Pull request merged**